core/tools/generate_names.cc

```diff
@@ -422,6 +422,14 @@ NameDef names[] = {
     {"export_", "export"},
     {"export_for_test"},
     {"restrict_to_service"},
+    {"sorbet"},
+    {"minTypedLevel", "min_typed_level"},
+    {"testsMinTypedLevel", "tests_min_typed_level"},
+    {"ignore"},
+    {"_false", "false"},
+    {"_true", "true"},
```

Comment:
```suggestion
    {"false_", "false"},
    {"true_", "true"},
```

Looking at the other names in this file, the convention is to put the underscore after the name when it would otherwise share a name with a C++ reserved word.